index.html

```diff
@@ -0,0 +1,244 @@
+          <h2 class="about-header mainheader">About Myself</h2>
+          <p class="about-text welcome-text">Hello I'm a software developer! I can help you build a product , feature or
+            website
+            Look through some of my work and
+            experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+          <h2 class="connect">LET'S CONNECT</h2>
+          <ul class="about-icons social-media">
+            <li class="media-icons">
+              <a href="#" target="_blank" rel="noopener noreferrer">
+                <img class="" src="images/github.svg" alt="Github">
+              </a>
+            </li>
+            <li class="media-icons">
+              <a href="#" target="_blank"
+                rel="noopener noreferrer">
+                <img class="s-m-icon" src="./images/Linkedin.svg" alt="Linkedin">
+              </a>
+            </li>
+            <li class="media-icons">
+              <a href="#" target="_blank" rel="noopener noreferrer">
+                <img class="" src="images/M.svg" alt="Angelist">
+              </a>
+            </li>
+            <li class="media-icons">
+              <a href="#" target="_blank" rel="noopener noreferrer">
+                <img class="" src="images/twiter.svg" alt="Twitter">
+              </a>
+            </li>
+            <li class="media-icons">
+              <a href="#" target="_blank" rel="noopener noreferrer">
+                <img class="" src="images/Vector3.svg" alt="Blog">
+              </a>
+            </li>
+          </ul>
+          <button class="about-btn button" type="button">Get my resume</button>
+        </div>
+        <ul class="skill-lists">
+          <li class="frames-lang-lists-container">
+            <div class="frames-lfs">
+              <h2 class="skill-title">Languages</h2>
+              <img src="./images/arrowdown.svg" alt="arow-dropdown-button">
+            </div>
+            <ul class="language-list">
+              <li class="language">
+                <img src="./images/javascript.svg" alt="JS">
+                <h2 class="lang-name">JavaScript</h2>
+              </li>
+              <li class="language">
+                <img src="./images/html5.svg" alt="HTML">
+                <h2 class="lang-name">HTML</h2>
+              </li>
+              <li class="language">
+                <img src="./images/css3.svg" alt="CSS">
+                <h2 class="lang-name">CSS</h2>
+              </li>
+            </ul>
+          </li>
+          <li class="frames-lfs">
+            <h2 class="skill-title">Frameworks</h2>
+            <img src="./images/arrowright.svg" alt="arow-dropdown-button">
+          </li>
+          <li class="line"></li>
+          <li class="frames-lfs">
+            <h2 class="skill-title">Skills</h2>
+            <img src="./images/arrowright.svg" alt="arow-dropdown-button">
+          </li>
+          <li class="line"></li>
```

Comment:
### OPTIONAL

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_169](https://user-images.githubusercontent.com/69638013/167262322-1300f47b-1993-462d-bf7f-0768e4924450.png)  |  ![Screenshot_168](https://user-images.githubusercontent.com/69638013/167262352-a732db65-eb4d-4ccc-ba93-e035e7b24819.png)

- [ ] Kindly add the right border color so your design looks exactly like the figma design 👍🏿